lte/cloud/go/services/lte/obsidian/models/gateway_pool_records_swaggergen.go

```diff
@@ -0,0 +1,45 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"strconv"
+
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/swag"
+)
+
+// GatewayPoolRecords gateway pool records
+// swagger:model gateway_pool_records
+type GatewayPoolRecords []*GatewayPoolRecord
```

Comment:
Ideally this wouldn't be a slice of ptrs. Makes for some weird/breakable constructs. Idk if there's some way to hack that in the swagger.yml